src/main/scala/akka/stream/contrib/Sample.scala

```diff
@@ -0,0 +1,74 @@
+package akka.stream.contrib
```

Comment:
We do. Missing header here was an indication that sbt did not see that file. And that was because it was not moved to the `contrib` dir. I created a PR to fix it: https://github.com/akka/akka-stream-contrib/pull/55